src/pages/Login/Login.scss

```
@@ -1 +1,69 @@
 @import '../../styles/variables.scss';
+@import url('https://fonts.googleapis.com/css2?family=Tangerine&display=swap');
+
+.login {
+  background-color: #ffffff;
+  @include flexCenter;
+
+  main {
+    .loginWrap {
+      @include flexCenter;
+      .bannerSlide {
+        img {
+          width: 350px;
+        }
+      }
+      .loginInner {
+        border: 1px solid lightgrey;
+        padding: 40px;
+        width: 350px;
+        height: 400px;
```

css selector가 시작될 때는 가독성을 위해 줄바꿈을 해주세요!
```suggestion
  main {
    .loginWrap {
      @include flexCenter;
      
      .bannerSlide {
        img {
          width: 350px;
        }
      }
      
      .loginInner {
        border: 1px solid lightgrey;
        padding: 40px;
        width: 350px;
        height: 400px;
```